etna/transforms/outliers.py

```diff
@@ -46,6 +46,12 @@ def fit(self, df: pd.DataFrame) -> "OutliersTransform":
         """
         ts = TSDataset(df, freq=pd.infer_freq(df.index))
         self.outliers_timestamps = self.detect_outliers(ts)
+        self.original_values = dict()
+
+        for segment, timestamps in self.outliers_timestamps.items():
```

Comment:
Better to create a separate method(`_save_original_values()`) for `self.original_values` initialization(imho)